tests/testthat/_snaps/unbump_version.md

```diff
@@ -0,0 +1,75 @@
+# unbump_version() works
+
+    Code
+      bump_version()
+    Message <cliMessage>
+      > Scraping 3 commit messages.
+      v Found 1 NEWS-worthy entries.
+      
+      -- Updating NEWS --
+      
+      > Adding new entries to 'NEWS.md'.
+      
+      -- Update Version --
+      
+      v Package version bumped to 0.0.0.9001.
+      > Adding header to 'NEWS.md'.
+      > Committing changes.
+      
+      -- Tagging Version --
+      
+      > Creating tag v0.0.0.9001 with tag message derived from 'NEWS.md'.
+    Message <rlang_message>
+      * Edit 'NEWS.md'
+    Message <cliMessage>
+      ! Call `fledge::finalize_version()`.
+    Output
+      NULL
+    Code
+      unbump_version()
+    Message <cliMessage>
+      i Checking if working copy is clean.
+      i Checking if last tag points to last commit.
+      i Checking if commit messages match.
+      v Safety checks complete.
+      > Deleting tag v0.0.0.9001.
+      v Resetting to parent commit 42.
+    Code
+      use_r("blop")
+    Message <rlang_message>
+      * Edit 'R/blop.R'
```

Comment:
I have used the quiet argument of `usethis::with_project()` so that's unexpected.